There is an `AttributeError` when `repr(querystring_parser)`

reproduce:

python: 3.8.5
python-multipart: 0.0.5

```python
import pytest
from multipart import QuerystringParser


# pylint: disable=invalid-name
def test_repr_QuerystringParser():
    parser = QuerystringParser(callbacks={})

    with pytest.raises(AttributeError):
        repr(parser)
```

details about that AttributeError

```
self = <[AttributeError("'QuerystringParser' object has no attribute 'keep_blank_values'") raised in repr()] QuerystringParser object at 0x7f171df674f0>

    def __repr__(self):
        return "%s(keep_blank_values=%r, strict_parsing=%r, max_size=%r)" % (
            self.__class__.__name__,
>           self.keep_blank_values, self.strict_parsing, self.max_size
        )
E       AttributeError: 'QuerystringParser' object has no attribute 'keep_blank_values'
```

--- a/multipart/multipart.py
+++ b/multipart/multipart.py
@@ -949,9 +949,9 @@ class QuerystringParser(BaseParser):
         self.callback('end')
 
     def __repr__(self):
-        return "%s(keep_blank_values=%r, strict_parsing=%r, max_size=%r)" % (
+        return "%s(strict_parsing=%r, max_size=%r)" % (
             self.__class__.__name__,
-            self.keep_blank_values, self.strict_parsing, self.max_size
+            self.strict_parsing, self.max_size
         )
 
 

--- a/multipart/tests/test_multipart.py
+++ b/multipart/tests/test_multipart.py
@@ -478,6 +478,10 @@ class TestQuerystringParser(unittest.TestCase):
             (b'another', b'asdf')
         )
 
+    def test_repr(self):
+        # Issue #29; verify we don't assert on repr()
+        _ignored = repr(self.p)
+
 
 class TestOctetStreamParser(unittest.TestCase):
     def setUp(self):

